eclipse/sw360
# Update the REST-API documentation for Definition of Manufacturer on project level

[//]: # (Copyright Bosch.IO GmbH 2020)
[//]: # (This program and the accompanying materials are made)
[//]: # (available under the terms of the Eclipse Public License 2.0)
[//]: # (which is available at https://www.eclipse.org/legal/epl-2.0/)
[//]: # (SPDX-License-Identifier: EPL-2.0)

> Please provide a summary of your changes here.
Rest-Docs not get updated for Definition of Manufacturer on project level issue so I write a junit code as per the exiting scenario and test it from rest api url. 
> * Which issue is this pull request belonging to and how is it solving it? (*Refer to issue here*)
> * Did you add or update any new dependencies that are required for your change?

Issue: #

### Suggest Reviewer
> You can suggest reviewers here with an @mention.

### How To Test?
1.login into sw360 
2. after login scroll down and click on REST-DOC
3. open chapter chapter [3.3.13]) and  chapter 3.3.2
4.we can see vendor is there in response.
> How should these changes be tested by the reviewer?
> Have you implemented any additional tests?

### Checklist
Must:
